sites/public/pages/listing/[id].tsx

```diff
@@ -0,0 +1,22 @@
+import Head from "next/head"
+import axios from "axios"
+
+export default function ListingRedirect(props: Record<string, string>) {
+  return (
+    <Head>
+      <meta httpEquiv="refresh" content={`0; url=${props.to}`} />
+    </Head>
+  )
+}
+
+export async function getServerSideProps(context: { params: Record<string, string> }) {
+  let response
+
+  try {
+    response = await axios.get(`${process.env.backendApiBase}/listings/${context.params.id}`)
```

Comment:
This is a general question about the use of axios. Will we want to pass headers to the server, and if so would it be better to create a reusable axios instance with those headers? Event if not, is there value in creating an instance for each API with a specified base URL like:
```
const listingApi = axios.create({
    baseUrl: `${process.env.backendApiBase}/listings/`,
   headers: {}
});
```
So then we can call `await listingApi(context.params.id)`.

Or at least set a global axios instance so we don't have to specify process.env.backendApiBase each time.

Reply:
Also we should for quite a bit of time be using an autogenerated frontend client (refer to how partners portal is e.g. fetching applications), no direct axios.get calls.

Reply:
I agree, let's refactor out the direct axios calls…not sure we should do it here since this PR is already getting massive.